### PR TITLE
Collapse nav on selection of nav item

### PR DESCRIPTION
**Pull request by readeral**
Trigger the nav toggle function on selecting a link

### CODE REVIEW

File: components/nav-bar/nav-bar.tsx [hunk 1]
```diff
@@ -72,7 +72,11 @@ const NavBar: FC<NavBarProps> = ({menuItems}) => {
                     width: ['auto', '75%']
                   }}
                 >
-                  <NavItems selectedMenu={selectedMenu} menuItems={menuItems} />
+                  <NavItems
+                    selectedMenu={selectedMenu}
+                    menuItems={menuItems}
+                    onNavigate={() => isOpen && toggleOpen()}
```
Comment (BarryThePenguin) [inline components/nav-bar/nav-bar.tsx]: An alternative is to use

```
import Router from 'next/router'

useEffect(() => {
  const handleRouteChange = () => isOpen && toggleOpen()

  Router.events.on('routeChangeStart', handleRouteChange)
  return () => {
    Router.events.off('routeChangeStart', handleRouteChange)
  }
}, [])
```

Comment (readeral) [reply components/nav-bar/nav-bar.tsx]: This is better right - rather than props-drilling?

Comment (BarryThePenguin) [reply components/nav-bar/nav-bar.tsx]: Not too worried about that. This more matches the behaviour we're after.

Close the nav overlay
- on click outside
- on route transition

**Comment by readeral** [comment]
Given this includes my other branch by accident, maybe we should close this one, and implement what you suggest. Would you be happy to do that?

**Pull request closed without merging**